doc/tutorial/examples.rst

```diff
@@ -347,15 +347,7 @@ afterwards compile this expression to get functions,
 using pseudo-random numbers is not as straightforward as it is in
 NumPy, though also not too complicated.
 
-The way to think about putting randomness into PyTensor's computations is
```

Comment:
This was a mess of an explanation, and we have a good guide explaining in `prng`.